app/assets/stylesheets/_headers_from_future_juntos.scss

```diff
@@ -425,3 +463,106 @@
 .u-marginbottom-40 {
   margin-bottom: 40px;
 }
+
+
+.container-content {
+    position: relative;
+    height: 100%;
+    width: 100%;
+    right: 0;
+    -webkit-transition:  right 0.4s ease-in-out;
+    -moz-transition:  right 0.4s ease-in-out;
+    -ms-transition:  right 0.4s ease-in-out;
+    -o-transition:  right 0.4s ease-in-out;
+    transition:  right 0.4s ease-in-out;
+}
+
+.container-content.open-sidebar {
+    right: 240px;
+}
+
+#sidebar {
```

Comment:
:+1: 
mas quando for fazer qualquer coisa do tipo use as features q o sass fornece pre você, vai facilitar sua e dos outros :)